art-blocks-api/artblocks-viewer.md

```diff
@@ -1,36 +1,65 @@
+
+- _Example:_ [Latest Artblocks_VO mint](https://live.artblocks.io/0x059edd72cd353df5106d2b9cc5ab83a52287ac3a)
+
+**live.artblocks.io/`{CONTRACT_ADDRESS}`/`{PROJECT_INDEX}`** enter any Art Blocks or Engine contract followed by the index of the project and you be presented with the latest _rendered_ mint.
+
+- _Example:_ [Latest Chromie Squiggle mint](https://live.artblocks.io/0x059edd72cd353df5106d2b9cc5ab83a52287ac3a/0)
+
+**live.artblocks.io/token/`{TOKEN_ID}`** enter any Art Blocks or Engine token id and you are presented with that token.
+
+- _Example:_ [Chromie Squiggle mint #71](https://live.artblocks.io/token/0x059edd72cd353df5106d2b9cc5ab83a52287ac3a-71)
+
+**live.artblocks.io/random/`{CONTRACT_ADDRESS}`** enter any Art Blocks or Engine contract address and you are presented with a random active project
+
+- _Example:_ [Random Art Blocks v3 contract](https://live.artblocks.io/random/0x99a9b7c1116f9ceeb1652de04d5969cce509b069)
 
-**live.artblocks.io/`{CONTRACT_ADDRESS}`** enter any Art Blocks or PBAB contract and you be presented with the latest *rendered* mint.  
-+ *Example:* [Latest Artblocks_VO mint](https://live.artblocks.io/0x059edd72cd353df5106d2b9cc5ab83a52287ac3a)
+**live.artblocks.io/random/`{CONTRACT_ADDRESS}/{PROJECT_INDEX}`** enter any Art Blocks or Engine contract address and a project index you are presented with a random token from that project
 
-**live.artblocks.io/`{CONTRACT_ADDRESS}`/`{PROJECT_INDEX}`** enter any Art Blocks or PBAB contract followed by the index of the project and you be presented with the latest *rendered* mint.   
-+ *Example:* [Latest Chromie Squiggle mint](https://live.artblocks.io/0x059edd72cd353df5106d2b9cc5ab83a52287ac3a/0)
+- _Example:_ [Random Chromie Squiggles](https://live.artblocks.io/random/0x059edd72cd353df5106d2b9cc5ab83a52287ac3a/0)
 
-**live.artblocks.io/token/`{TOKEN_ID}`** enter any Art Blocks or PBAB token id and you are presented with that token.   
-+ *Example:* [Chromie Squiggle mint #71](https://live.artblocks.io/token/0x059edd72cd353df5106d2b9cc5ab83a52287ac3a-71)
+## Random Query Params
+
+You can add any query params from the customizable display listed below as well as two random specific query params.
+
+- **reloadClick**: this is to turn on click reloading where if you click the token it will give you a new token from that project. `default: false`
+- **reloadDelay**: this is to turn on auto refreshing after the specified time in seconds to give you a new token from that project. `default: none`
+
+**Example URLs**:
+
+- [Click turned on](https://live.artblocks.io/random/0x059edd72cd353df5106d2b9cc5ab83a52287ac3a/0?reloadClick=true)
+- [Delay of 30 seconds and click turned on](https://live.artblocks.io/random/0x059edd72cd353df5106d2b9cc5ab83a52287ac3a/0?reloadClick=true&reloadDelay=30)
+- [Delay of 30 seconds with no click](https://live.artblocks.io/random/0x059edd72cd353df5106d2b9cc5ab83a52287ac3a/0?reloadDelay=30)
 
 ## Contracts
```

Comment:
Would remove this, assuming that any AB Ecosystem contract could be used, no need to keep a manually updated list here